api/volume.go

```diff
@@ -127,6 +129,18 @@ func (s *Server) VolumeCreate(rw http.ResponseWriter, req *http.Request) error {
 		}
 	}
 
+	if volume.AccessMode == "" && volume.FromBackup != "" {
```

Comment:
Can it be added to the mutating webhook?

Reply:
Thanks, I will follow the suggestion to update access mode value into backup label through mutating webhook `webhook/resources/backup/mutator.go`.